view/frontend/web/js/view/payment/method-renderer/cc.js

```diff
@@ -37,7 +37,9 @@ define([
                         contractOptions.push(item);
                     }
                 });
-                contractOptions.push({contractId:0,label:$t('Use Different Card')});
+                if (contractOptions.length){
```

Comment:
SHould this be `.length > 0` or osmething like that?

Reply:
Similar syntax is used above on line 32 `window.checkoutConfig.payment.reach_cc.open_contracts.length`